tests/kernel/mem_protect/userspace/src/main.c

```diff
@@ -1171,6 +1205,13 @@ void test_main(void)
 			 ztest_unit_test(test_stack_buffer),
 			 ztest_user_unit_test(test_unimplemented_syscall),
 			 ztest_user_unit_test(test_bad_syscall),
+#if defined(CONFIG_ARM) || defined(CONFIG_X86)
```

Comment:
eh, put an #ifndef CONFIG_ARC and leave a comment /* FIXME: #17590 */

Reply:
Fixed